betterrolls-swade2/scripts/global_actions.js

```diff
@@ -194,7 +194,7 @@ function check_selector(type, value, item, actor) {
   } else if (type === "actor_name") {
     selected = actor.name.toLowerCase().includes(value.toLowerCase());
   } else if (type === "actor_has_item") {
-    const ITEM_TYPES = ["weapon", "armor", "shield", "gear", "consumable"];
+    const ITEM_TYPES = ["skill", "weapon", "armor", "shield", "gear", "consumable"];
```

Comment:
This part of the change should be unnecessary. There is already a skill selector type. The actor_has_item selector refers to physical items not the class of objects stored in actor.items.

Reply:
it does have a skill selector, but the selector isn't checking the actor or token to see if it has the skill. That selector is to check if the item is a skill. This selector checks if the actor or token has the skill which is a type of item. I'm running it now, and it works great.

Reply:
I'll let @javierriveracastro make the final call, but in that case, we should be adding an actor_has_skill selector, not just adding skills to actor_has_item.

Reply:
It works, but I agree that it is confusing. A skill is not an item in casual speak, not in SW rules. It is an item to Foundry, but a power, or and edge, is also an item.

I feel that this makes it confusing. A new selector actor_has_skill is preferable, but it si also more work.

Reply:
let me see what it would take to add it. I've got some time.